main.py

```diff
@@ -79,6 +79,15 @@ async def getTheme(ctx):
   else:
     await ctx.message.reply("default")
 
+@client.command()
+async def setStatus(ctx,arg='Online'):
```

Comment:
Thanks for the PR @afchacon! Could you also add this new command to the `help` command so users are aware of its presence?